app-admin/audit/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+abinfo "Using python3 as python intepreter ..."
+export PYTHON=python3
```

Comment:
This might cause problems in the resulting shebang, better to specify as /usr/bin/python3.

Reply:
Done